MFVNeutralino/test/TrackMoverHighEta/SampleFiles.py

```diff
@@ -0,0 +1,1036 @@
+import os, base64, zlib, cPickle as pickle
```

Comment:
I don't think we need a second SampleFiles.py, this seems _really_ hard to keep synchronized with the main one.

Reply:
Remove it. Done.